python/ql/src/Security/CWE-312/CleartextLogging.ql

```diff
@@ -22,5 +22,5 @@ from Configuration config, DataFlow::PathNode source, DataFlow::PathNode sink, s
 where
   config.hasFlowPath(source, sink) and
   classification = source.getNode().(Source).getClassification()
-select sink.getNode(), source, sink, "$@ is logged here.", source.getNode(),
-  "Sensitive data (" + classification + ")"
+select sink.getNode(), source, sink, "This logs sensitive data returned by $@ as clear text.",
```

Comment:
The final message would contain `sensitive data` twice.. and there's no reason to believe that the source is a call. So my suggestion would be

```diff
-select sink.getNode(), source, sink, "This logs sensitive data returned by $@ as clear text.",
+select sink.getNode(), source, sink, "This logs $@ as clear text.",
```

combining with @tausbn's suggestion:

```suggestion
select sink.getNode(), source, sink, "This expression logs $@ as clear text.",
```